docs/pages/server-access/guides/openssh/openssh.mdx

```diff
@@ -318,3 +318,19 @@ $ ssh -F ssh_config_teleport ${USER?}@node2.leafcluster.${CLUSTER}
   qualified domain name, rather than an IP address.
 
 </Admonition>
+
+### Upgrading to Teleport 14 and migrating from legacy OpenSSH nodes
```

Comment:
I would move this note to the Prerequisites section so a user doesn't follow this guide with Teleport 14 only to end up disappointed

Reply:
Done